lib/jekyll/liquid_renderer/table.rb

```diff
@@ -75,18 +80,23 @@ def data_for_table(num_of_rows)
         sorted = @stats.sort_by { |_, file_stats| -file_stats[:time] }
         sorted = sorted.slice(0, num_of_rows)
 
-        table = [%w(Filename Count Bytes Time)]
+        gauges = [:count, :bytes, :time]
+        labels = %w(Filename).concat(gauges.map { |gauge| gauge.to_s.capitalize })
+        table  = [labels]
+        total  = Hash.new { |hash, key| hash[key] = 0 }
 
         sorted.each do |filename, file_stats|
-          row = []
-          row << filename
-          row << file_stats[:count].to_s
-          row << format_bytes(file_stats[:bytes])
-          row << format("%.3f", file_stats[:time])
-          table << row
+          gauges.each { |gauge| total[gauge] += file_stats[gauge] }
+          table << data_to_row(file_stats, [filename])
         end
 
-        table
+        table << data_to_row(total, ["TOTAL (for #{sorted.size} files)"])
+      end
+
+      def data_to_row(hash, row)
```

Comment:
We're not really formatting anything here, we're adding data. I think a method rename makes sense here:
```suggestion
      def add_data(hash, row)
```